src/qtcore/qml/QMLEngine.js

```diff
@@ -127,10 +127,22 @@ QMLEngine = function (element, options) {
 
         this.start();
 
-        // Call completed signals
+        // #engine [engine::loadQMLTree] -> [engine::callCompletedSignals]
+        this.callCompletedSignals();
+    }
+
+    this.callCompletedSignals = function () {
```

Comment:
Since this is a new function, why not create it in `QMLEngine.prototype` from the start?